packages/elements/src/photon-multirx-form/components/AddPrescriptionCard.tsx

```diff
@@ -170,6 +175,8 @@ export const AddPrescriptionCard = (props: {
           header: 'Prescription Added',
           body: 'You can send this order or add another prescription before sending it'
         });
+
+        props.screenDraftedPrescriptions();
```

Comment:
let's run a screening here when we add a drafted prescription to an order

Reply:
I thought the screen would happen at the time of selecting the medication and template so that it would show inline with the form, right?

Reply:
<img width="607" alt="Screen Shot 2024-10-11 at 12 48 29 PM" src="https://github.com/user-attachments/assets/fde2d1ad-88b6-4c80-949d-96f0d01901cc">
if we did it here, we'd already have the info

Reply:
you're right - we do do this and there's no need to make a second request here since the alerts will still apply until someone changes the prescription drafter medication